### ISSUE TITLE
Always default to downloading the Windows version of a game

### ISSUE THREAD

**Issue by CommandMC** [Heroic-Games-Launcher/HeroicGamesLauncher#4089]
### Problem description

Epic and GOG both feature macOS versions of some games, and GOG additionally has Linux versions. Sadly, many of these versions have been long abandoned and no longer work on modern systems. Using the Windows version through Proton/Wine/CrossOver/etc. often works better.

### Feature description

Instead of defaulting to the native platform (macOS if on macOS, Linux if on Linux), always suggest to install the Windows version of a game first.

**Comment by arielj** [comment]
while I did notice the windows version usually works better on linux, I also noticed the mac version usually works better on mac

so I'd suggest make this ticket only apply for linux and not for mac

**Comment by shortthu** [comment]
Making this a toggle on settings would be sufficient enough I think

**Comment by iFloneUEFN** [comment]
@shortthu I do agree.

**Comment by aserebryakov** [comment]
I agree with the issue because GOG games that are linux native do not support cloud saves, that is absolute must.

**Comment by PimpOfSune** [comment]
I'd REALLY appreciate some statistics.

There's slightly more than 1000 games in my libraries across multiple launchers.
Of games which have Linux support declared, yet broken, I can list exactly 4 (four).
ADOM (GOG)
Victor Vran (GOG)
Project Highrise (Steam)
Alicia Quatermain 1 (Steam)
(in three cases out of four it's entirely developers' fault BTW)

It's easier to flip target platform when installing four specific games rather than all except four.

**Comment by imLinguin** [comment]
The only native games come from GOG. And unfortunately GOG and devs treat these builds as second class citizens. 
GOG doesn't specify cloud save config for Linux, most games don't ship Galaxy SDK - no achievements and multiplayer.
The change comes to default selection anyway, if you really want a native build you can always select it when installing

**Comment by PimpOfSune** [comment]
If I want native build, under such circumstances, I can use DRM-free installer without Heroic then.

> The only native games come from GOG. And unfortunately GOG and devs treat these builds as second class citizens.

Once again, 2 (two) games out of... let me check in a jiffy... 120ish?

> GOG doesn't specify cloud save config for Linux

I believe it's a part of a larger problem with common Linux distros drifting towards Android-like behavior with software's auxiliary files.
It should be resolved by some other means.
Solutions like mtkennerly/ludusavi can help with that, if it's SUUUUUUUUCH a problem.

> no achievements and multiplayer

Well, yes.
Because these are parts of DRM, you know.
And №1 use case for GOG games (most talked about, at the very least) is "DRM-free gaming".

Multiplayer is a bit more serious problem than it looks though - still, is it worth resorting to brute force solutions like that?

I have no intent to persuade someone, especially since damage was done already.
Just warn.
After all, it won't be me who'll handle all the resulting feedback after v2.19 release.

**Comment by TheSunCat** [comment]
I work for a game studio that actually maintains their Linux native builds and puts quite some effort into it, it is a little disappointing to see people get defaulted to the Windows version of our game.
Would it be a good compromise to prompt players when a Linux version is available, so they can make their own informed choice?

**Comment by aserebryakov** [comment]
> I work for a game studio that actually maintains their Linux native builds and puts quite some effort into it, it is a little disappointing to see people get defaulted to the Windows version of our game. Would it be a good compromise to prompt players when a Linux version is available, so they can make their own informed choice?

From my use case cloud saves are one of the essential things. I played "Slay The Princess" native and didn't backup them, so I lost my first playthrough when deleted game and installed Windows version.
